content/rest-api/teams.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Teams API
+description: API for managing team members
+
+weight: 5
+---
+
+This document describes the API endpoints for managing teams. To use this API, you must be a **team owner**.
+
+
+>Read more about the [Teams](../getting-started/teams/) feature, the available user roles and permissions.
+
+
+
+## Invite a new team member
+Invite a new team member to your team.
+
+`POST /:team_id/invitation`
+
+#### Parameters
+
+
+| **Name** | **Type** | **Description**                                            |
+|----------| -------- |------------------------------------------------------------|
+| `email`  | `string` | **Required.** SSH or HTTPS URL for cloning the repository. |
+| `role`   | `string` | **Required.** Could be `owner` or `developer`              |
+
+#### Example
+
+{{< highlight bash "style=paraiso-dark">}}
+curl -H "Content-Type: application/json" \
+     -H "x-auth-token: <API Token>" \
+     -d '{
+        "email": "your_user@domain.com",
+        "role": "developer"
+     }' \
+     -X POST https://api.codemagic.io/<team_id>/invitation
+{{< /highlight >}}
+
+
+#### Response
+
+A full team object is returned. You can see that your invitation has been created
+in the `invitations` section.
+Also, you can see your current active team members in the `collaborators` section,
+ where you can locate their `user_ids`.
+
+
+{{< highlight json "style=paraiso-dark">}}
+{
+  "team":{
+    "_id":"<team_id>",
+    "applicationIds":[...],
+    "billing":{...},
+    "buildTimes": {...},

```

Comment:
let's keep only `invitations` and I don't see if we return `id` in invitations

Reply:
No, I think user is not created already, only the invitation.

Reply:
ah I see. Can you test if invitation endpoint works correctly with `owner` role?
if it works as described, we can completely remove `change a team member role` section and this example. Sorry for the late idea

Reply:
yes, you can invite a member as developer or as owner and work as expected